doc/content/examples/index.md

```diff
@@ -17,5 +17,7 @@ A list of example simulations using MASTODON is provided below. These examples i
 - [Example 7](examples/example7.md): Demonstration of domain reduction method.
 - [Example 8](examples/example8.md): Seismic risk assessment of a generic nuclear facility using the MASTODON FTA Python module.
 - [Example 9](examples/example9.md): Soil-structure interaction analysis of an undamped rigid structure on undamped linear soil.
+- [Example 10a](examples/example10a.md): Cantilever example demonstrating central difference time integration.
```

Comment:
Consider adding "using HEX8 elements"  in the end.

Reply:
HEX8 elements are default, so I don't think we need to mention that. I'll merge it as is for now..thanks!